docs/cloud-native-security/cloud-nat-sec-posture-dashboard.asciidoc

```diff
@@ -24,10 +24,17 @@ The Cloud Posture dashboard shows:
 [[cloud-nat-sec-posture-dashboard-UI]]
 === Cloud Posture dashboard UI
 
-The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the CSP of an individual Kubernetes cluster.
+The top section of the dashboard summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Three summary cards on the left of this section show Clusters Evaluated, Resources Evaluated, and Failed Findings. You can click *Resources Evaluated* or *Failed Findings* to open the <<findings-page, Findings page>> grouped by resources or filtered for failed findings, respectively. The remaining summary cards show your overall compliance score, and total failed findings grouped by CIS section. Click any CIS section name to view failed findings from that section on the <<findings-page, Findings page>>.
 
-The Cloud Posture Score card shows the collective performance of all your monitored clusters. Hover over the card to display when the data was collected.
+Below the summary section, each row shows the CSP of an individual Kubernetes cluster, including the cluster's name, when its posture was last evaluated, its compliance score, and failed findings grouped by CIS section. When you begin to monitor a new cluster, a new row appears.
+
+[discrete]
+[[cloud-nat-sec-posture-dashboard-faq]]
+=== FAQ
```

Comment:
We have to think about if our non-native English speakers will know what FAQ stands for. 
```suggestion
=== FAQ (Frequently asked questions) 
```

Reply:
I would consider putting these in a drop-down, especially if we're going to add to this section in the future. Example in this doc [here](https://www.elastic.co/guide/en/security/current/ts-detection-rules.html). Not critical, but something to think about. 

```
.title
[%collapsible]
====
text text text
====
```